paasta_tools/check_chronos_jobs.py

```diff
@@ -292,12 +298,19 @@ def main():
             except utils.NoDeploymentsAvailable:
                 paasta_print(utils.PaastaColors.cyan("Skipping %s because no deployments are available" % service))
                 continue
+            job_expected_runtime = 0
+            if chronos_job:
+                try:
+                    job_expected_runtime = int(client.job_stat(chronos_job['name'])['histogram']['99thPercentile'])
```

Comment:
I just realized that this request is redundant when last_run_time + interval is more than datetime.now (i.e. the job hasn't started yet).
However this implementation is faster than the current one anyway.
I'd rather ship it as it is, and then refactor this part in a new PR.